addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -94,8 +94,15 @@
 
                         <t t-set="current_subtotal" t-value="current_subtotal + line.price_subtotal"/>
 
-                        <tr t-att-class="'fw-bold o_line_section' if line.display_type == 'line_section' else 'fst-italic o_line_note' if line.display_type == 'line_note' else ''">
-                            <t t-if="not line.display_type">
+                        <tr
+                            t-att-class="'fw-bold o_line_section' if (
+                                line.display_type == 'line_section'
+                                or line.product_type == 'combo'
+                            )
+                            else 'fst-italic o_line_note' if line.display_type == 'line_note'
+                            else ''"
+                        >
+                            <t t-if="not line.display_type and line.product_type != 'combo'">
```

Comment:
Dunno if you considered this approach, but would it help to introduce a combo `display_type` (which would still required a product though)

(Not for this PR, but such a change could be done before 18 if we consider it worthwhile)

Reply:
Yes, I thought about doing this but didn't investigate too much due to the urgency of this PR. I might create a POC later on.